os/booting-on-azure.md

```diff
@@ -16,19 +16,19 @@ The following command will create a single instance. For more details, check out
     <div class="tab-pane" id="alpha">
       <div class="channel-info">
         <p>The Alpha channel closely tracks master and is released frequently. The newest versions of system libraries and utilities will be available for testing. The current version is Container Linux {{site.alpha-channel}}.</p>
-        <pre>azure vm create --custom-data=config.ign --vm-size=Small --ssh=22 --ssh-cert=path/to/cert --no-ssh-password --vm-name=node-1 --location="&lt;location&gt;" my-cloud-service $(azure vm image list --json | jq --raw-output '.[].name | select(contains("__CoreOS-Alpha-{{site.alpha-channel}}"))') core</pre>
+        <pre>az vm create --name node-1 --resource-group group-1 --admin-username core --custom-data config.ign --image CoreOS:CoreOS:Alpha:{{site.alpha-channel}}</pre>
```

Comment:
Does `az` use the image labels that allow you to omit the version number? Ideally, we'd use those and just specify that users should run the latest Alpha rather than a specific version which happens to be the latest.